Mlem/Views/Tabs/Feeds/Feed Root.swift

```diff
@@ -34,7 +34,9 @@ struct FeedRoot: View {
                     FeedView(
                         community: rootDetails.community,
                         feedType: rootDetails.feedType,
-                        sortType: defaultPostSorting,
+                        sortType: rootDetails.community != nil
+                        ? feedSortTypeTracker.getSortType(for: rootDetails.community!)
+                        : feedSortTypeTracker.getSortType(for: rootDetails.feedType),
```

Comment:
With the protocol approach suggested above, `getSortType` would be the same for either type, so this could collapse to:

`sortType: feedSortTypeTracker.getSortType(for: rootDetails.community ?? rootDetails.feedType)`